src/index.html

```diff
@@ -1,13 +1,72 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        src="images/imac.jpeg"
+        alt="imac"
+        class="card__img"
+      >
+      <div class="card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </div>
+      <div class="card__code">
+        Код товара: 195434
+      </div>
+      <div class="card__review">
+        <div class="card__rate">
+          <img class="card__star-img"
+            src="images/star-active.svg"
+            alt="star-active"
+          >
+          <img class="card__star-img"
+            src="images/star-active.svg"
+            alt="star-active"
+          >
+          <img class="card__star-img"
+            src="images/star-active.svg"
+            alt="star-active"
+          >
+          <img class="card__star-img"
+            src="images/star-active.svg"
+            alt="star-active"
+          >
+          <img class="card__star-img"
+            src="images/star.svg"
+            alt="star"
+          >
```

Comment:
In our slack chat there was a suggestion on how to reduce the length of html code in this exact chunk and make it a bit easier to adjust when it is needed in larger project. The way you do that is by importing the stars using CSS property 'background-image', adding a common class for all the stars and then making a modifier for this class.  Check the thread underneath this message if you're interested(http://joxi.ru/brRxOGWtL30pD2).
However, I don't think it is crucial.